Corona-Warn-App/src/main/res/values/srs_submission_strings.xml

```diff
@@ -1,4 +1,14 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources xmlns:tools="http://schemas.android.com/tools" tools:ignore="MissingTranslation">
 
+    <!-- Home screen register test card -->
```

Comment:
are the old strings still used? can we delete them?

Reply:
Yes